README.md

```diff
@@ -1 +1,96 @@
-# portal-lang
\ No newline at end of file
+# Cleo Portal Localization Tool
+
+Cleo Portal is localized for the United States out of the box, but other custom localizations can be created and added to the application after it is installed.
+
+Cleo Portal uses [gettext](https://en.wikipedia.org/wiki/Gettext) for localization.
+
+The application ships with a POT file (gettext template) containing text strings in American English.
+The POT file can be translated using a gettext translation editor to produce PO files (gettext translations) for other languages.
+
+However, Cleo Portal cannot read PO files, so they must be compiled to JSON files to be used in the application.
+
+This localization tool compiles PO files to JSON files.
+
+## Prerequisites
+
+- You must have [Git](https://git-scm.com/) installed
+- You must have [Node.js](https://nodejs.org/) installed, preferably the latest LTS version
+- You will need a gettext translation editor, such as the free [Poedit](https://poedit.net/)
+
+It is assumed that you have at least a basic understanding of how to use Git and npm (Node.js package manager) from the command line.
+
+It is also assumed that you know how to use Poedit or something similar. However, Poedit is easy to figure out even if you have never used it before.
+
+## Setup
+
+Clone the Git repo:
+
+```shell
+$ git clone https://github.com/CleoDev/portal-localization
+```
+
+Install the localization tool's dependencies (run from within the `portal-localization` folder):
```

Comment:
I think the npm installation and `Usage` instructions would be better served as sub text under number 5 of the `Localizing Cleo Portal` section